sdk/src/mlnotify/logger.py

```diff
@@ -2,24 +2,39 @@
 import logging.config
 
 LOGGER_NAME = "mlnotify"
+is_logger_initiated = False
 
-logging.config.dictConfig(
-    {
-        "version": 1,
-        "handlers": {
-            "console": {
-                "class": "logging.StreamHandler",
-                "level": "WARNING",
-                "formatter": "standard",
-                "stream": "ext://sys.stdout",
-            }
-        },
-        "formatters": {"standard": {"format": "%(asctime)s [%(name)s] %(levelname)s: %(message)s"}},
-        LOGGER_NAME: {"level": "WARNING", "handlers": ["console"]},
-        "disable_existing_loggers": False,
-    }
-)
 
-logger = logging.getLogger(LOGGER_NAME)
+def init_logger():
+    global is_logger_initiated
+    logging.config.dictConfig(
+        {
+            "version": 1,
+            "handlers": {
+                "console": {
+                    "class": "logging.StreamHandler",
+                    "level": "WARNING",
+                    "formatter": "standard",
+                    "stream": "ext://sys.stdout",
+                }
+            },
+            "formatters": {"standard": {"format": "%(asctime)s [%(name)s] %(levelname)s: %(message)s"}},
+            LOGGER_NAME: {"level": "WARNING", "handlers": ["console"]},
+            "disable_existing_loggers": False,
+        }
+    )
+    is_logger_initiated = True
+    logger = logging.getLogger(LOGGER_NAME)
 
-logger.debug(f"Logger is set to level {logger.level}")
+    logger.debug(f"Logger is set to level {logger.level}")
+    return logger
+
+
+def get_logger():
```

Comment:
There's no need for lazy initialization, I just wanted the logic moved to a function to make it cleaner and easier to mock in tests

Reply:
Done.